packages/flutter/lib/src/widgets/overscroll_indicator.dart

```diff
@@ -73,6 +74,48 @@ import 'ticker_provider.dart';
 /// }
 /// ```
 /// {@end-tool}
+///
+/// Instead of switching to [NestedScrollView], the position where the glow is
+/// drawn can also be manipulated by making use of a
+/// [NotificationListener] and manipulating the
+/// [OverscrollIndicatorNotification.paintOffset] value.
+/// This can be used to place the glow at an arbitrary offset in case the
+/// problem cannot be solved easily with a [NestedScrollView].
+///
+/// {@tool dartpad --template=stateless_widget_scaffold}
+///
+/// This example demonstrates how to use a [NotificationListener] to manipulate
+/// the placement of a [GlowingOverscrollIndicator] when building a
+/// [CustomScrollView]. Drag the scrollable to see the bounds of the overscroll
+/// indicator.
+///
+/// ```dart
+/// Widget build(BuildContext context) {
+///   double leadingPaintOffset = MediaQuery.of(context).padding.top + AppBar().preferredSize.height;
+///   return NotificationListener<OverscrollIndicatorNotification>(
+///     onNotification: (notification) {
+///       if (notification.leading) {
+///         notification.paintOffset = leadingPaintOffset;
+///       }
+///       return false;
+///     },
+///     child: CustomScrollView(
+///       slivers: [
+///         SliverAppBar(title: Text('Custom PaintOffset')),
+///         SliverToBoxAdapter(
+///           child: Container(
+///             color: Colors.amberAccent,
+///             height: 100,
+///             child: Center(child: Text('Glow all day!')),
+///           ),
+///         ),
+///         SliverFillRemaining(child: FlutterLogo()),
+///       ],
+///     ),
+///   );
+/// }
+/// ```
+/// {@end-tool}
```

Comment:
After this, let's link to some of the elements you have referenced here, like `OverscrollIndicatorNotification` and `NotificationListener`. There is a `See also: ...` format that is used in other classes.

Reply:
Done.